projects/zlib.net/package.yml

```diff
@@ -9,6 +9,9 @@ build:
   script: |
     ./configure --prefix="{{prefix}}"
     make --jobs {{ hw.concurrency }} install
+  env:
+    linux/aarch64:
+      CFLAGS: '$CFLAGS -fPIC'
```

Comment:
This is what we do in brewkit, should we be doing this for all Linux then? I dunno.

![image](https://github.com/pkgxdev/pantry/assets/58962/f46b912d-bb34-4e50-bed7-db1e80011c07)

Reply:
i think yes. it's non-harmful, minor performance cost, and many things dislike non-aligned binaries.